templates/fsharp/xplat/AvaloniaTest.Browser/Program.fs

```diff
@@ -18,6 +19,6 @@ module Program =
     let main argv =
         buildAvaloniaApp()
             .UseReactiveUI()
-            .SetupBrowserApp("out")
-            |> ignore
+            .StartBrowserAppAsync("out")
+            .Result
```

Comment:
Is it possible to return Task here without awaiting or blocking anything?
Blocking async calls isn't supported on WASM backend.

Reply:
Yes, a better way is needed here, will take another look later (I am not to familiar with F#) or someone with better F# knowledge can give some hint

Reply:
I don't think there is needed for better way. It just needs return a Task produced by StartBrowserAppAsync.
This code is executed by WASM interop layer, so it is expected to either return int/void or return a Task.

Reply:
Thanks to ChatGPT I found out how to do it in F#. It would be a bit nicer though if StartBrowserAppAsync would return `Task<int>` instead of `Task`